tests/dashbio_demos/app_volcano_plot.py

```diff
@@ -65,213 +65,212 @@ def header_colors():
+                        html.Div(
+                            id='vp-num-points-display',
+                            children=[
+                                html.Div(
+                                    title='Number of points in the upper left',
+                                    children=[
+                                        daq.LEDDisplay(
+                                            className='vp-input-like',
+                                            label='Upper left points',
+                                            id='vp-upper-left',
+                                            size=25,
+                                            color='#19D3F3'
+                                        ),
+                                        html.Div(
+                                            className='vp-test-util-div',
+                                            id='vp-upper-left-val'
+                                        )
+                                    ]
+                                ),
+                                html.Br(),
+                                html.Div(
+                                    className='vp-vertical-style',
+                                    title='Number of points in the upper right',
+                                    children=[
+                                        daq.LEDDisplay(
+                                            className='vp-input-like',
+                                            label='Upper right points',
+                                            id='vp-upper-right',
+                                            size=25,
+                                            color='#19D3F3'
+                                        ),
+                                        html.Div(
+                                            className='vp-test-util-div',
+                                            id='vp-upper-right-val'
+                                        )
+                                    ]
+                                ),
+                            ],
+                        ),
+                        html.Hr(),
+                        html.Div(id='vp-event-data')
+                    ])
+                )
+            ])
+        ])
+    ])
 
 
 def callbacks(app):  # pylint: disable=redefined-outer-name
+    @app.callback(
+        Output('vp-event-data', 'children'),
+        [Input('vp-graph', 'hoverData'),
+         Input('vp-graph', 'clickData')]
+    )
+    def get_event_data(hover, click):
+        hover_data_div = [
+            html.Div(className='vp-option-name', children='Hover data')
+        ]
+        hover_data = 'Hover over a data point to see it here.'
+
+        if hover is not None:
+            hovered_point = hover['points'][0]
+            hover_data = [
+                'x: {}'.format(hovered_point['x']),
+                html.Br(),
+                'y: {}'.format(hovered_point['y']),
```

Comment:
Could we keep fewer decimals for the y coordinate?
![y_decimals](https://user-images.githubusercontent.com/2227806/56128807-f7d69a80-5f80-11e9-8795-890a04f7f63b.png)

Reply:
Fixed in 7e90265